README.md

```diff
@@ -1,20 +1,20 @@
-# Borg cron helper scripts with local lock system
+# Borg cron helper scripts
 
-**Automate backups with borg in a more convenient and reliable way!**
-
-This scripts are some small and handy shell scripts to automate the backup process with [BorgBackup](https://borgbackup.readthedocs.io/). They are POSIX-compatible, so they should run with all shells. You're free to modify them for your needs!
+These scripts are some small and handy shell scripts to automate the backup process with [BorgBackup](https://borgbackup.readthedocs.io/). They are POSIX-compatible, so they should run with all shells. You're free to modify them for your needs!
 
 They add some convienent features around borg, regarding environments with only **one client**.
 
+The local lock system cirumvents the issue of [stale](https://github.com/borgbackup/borg/issues/813) [lock files](https://github.com/borgbackup/borg/issues/2306).
+
+## 
 ## Local lock
-The local lock system cirumvents the issue of [stale](https://github.com/borgbackup/borg/issues/813) [lock files](https://github.com/borgbackup/borg/issues/2306). When the backup process is interrupted, it can happen, that the remote borg repository stays locked. That's why further backups will fail.
-Obviously this is bad for an automated system, especially when talking about backup software.
 
-The issue [has been addressed](https://github.com/borgbackup/borg/pull/1674) in borg **v1.1.0** (currently beta), but I have not tested it and until it works there, here is my workaround.
+When the backup process is interrupted, sometimes the remote borg repository stays locked. That's why further backups will fail.
 
+The issue [has been addressed](https://github.com/borgbackup/borg/pull/1674) in borg **v1.1.0** (currently beta), but I have not tested it and until it works there, here is my workaround:
```

Comment:
No `:` necessary. This is continuous text, there is no bullet point afterwards or similar, so… leave it away.